src/Controls/src/Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -71,33 +71,52 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 			return config;
 		}
 
-		/// <summary>Bindable property for attached property <c>UseSafeArea</c>.</summary>
+		/// <summary>
+		/// Controls whether padding values are overridden with the safe area insets.
+		/// </summary>
 #if MACCATALYST
 		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create("UseSafeArea", typeof(bool), typeof(Page), true);
 #else
 		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create("UseSafeArea", typeof(bool), typeof(Page), false);
 #endif
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='GetUseSafeArea']/Docs/*" />
+		/// <summary>
+		/// Gets a value that indicates whether padding values are overridden with values that conform to the safe area on the device.
+		/// </summary>
+		/// <param name="element">The element whose safe area behavior to get.</param>
+		/// <returns>A Boolean value that tells whether padding values are overridden or not.</returns>
```

Comment:
Since we're returning a boolean we should try to specify what the `true` or `false` results mean https://github.com/dotnet/dotnet-api-docs/wiki/Returns:-Methods

Reply:
Applied changes.